lib/radiator_web/controllers/api/chapters_controller.ex

```diff
@@ -55,4 +58,17 @@ defmodule RadiatorWeb.Api.ChaptersController do
         |> halt()
     end
   end
+
+  defp assign_chapter(conn, _) do
+    with {:ok, chapter} <-
+           Editor.get_chapter(current_user(conn), conn.assigns[:audio], conn.params["start"]) do
+      conn
+      |> assign(:chapter, chapter)
+    else
+      error = {:error, _} ->
+        conn
+        |> RadiatorWeb.Api.FallbackController.call(error)
```

Comment:
Hmm, shouldn't we somehow just refer to the assigned fallbackcontroller? e.g. something like 
```elixir
apply(@phoenix_fallback, :call, [error])
```
? - in the middle of some other change now, can't verify if that is the way this should work. but the falback seems to be in the phoenix_fallback attribute